neuromation/cli/formatters/admin.py

```diff
@@ -13,3 +15,12 @@ def __call__(self, clusters_users: Iterable[_ClusterUser]) -> List[str]:
             rows.append((user.user_name, user.role.value))
 
         return list(table(rows=rows))
+
+
+class ClustersFormatter:
+    def __call__(self, clusters: Iterable[_Cluster]) -> List[str]:
+        headers = [click.style("Name", bold=True)]
```

Comment:
Not a problem for 1 row, but if we add several we hit alignment issues. Created https://github.com/neuromation/platform-client-python/issues/1245 for that.